backend/src/nginx/main-ui.conf.template

```diff
@@ -1,4 +1,8 @@
 
+map $http_upgrade $connection_upgrade {{
+        default upgrade;
+        '' $http_connection;
+}}
```

Comment:
just curious what this is doing?

Reply:
this only sets the `Connection` header to `upgrade` if the `Upgrade` header is set.